src/langgraph-platform/use-remote-graph.mdx

```diff
@@ -1,289 +1,300 @@
 ---
-title: How to interact with the deployment using RemoteGraph
-sidebarTitle: Interact with the deployment using RemoteGraph
+title: How to interact with a deployment using RemoteGraph
+sidebarTitle: Interact with a deployment using RemoteGraph
 ---
-<Info>
-**Prerequisites**
-* [LangGraph Platform](/langgraph-platform/index)
-* [LangGraph Server](/langgraph-platform/langgraph-server)
-</Info>
 
-`RemoteGraph` is an interface that allows you to interact with your LangGraph Platform deployment as if it were a regular, locally-defined LangGraph graph (e.g. a `CompiledGraph`). This guide shows you how you can initialize a `RemoteGraph` and interact with it.
+[`RemoteGraph`](https://langchain-ai.github.io/langgraph/reference/remote_graph) is a client-side interface that allows you to interact with your [deployment](/langgraph-platform/deployment-options) as if it were a local graph. It provides API parity with [`CompiledGraph`](/oss/python/langgraph/graph-api#compiling-your-graph), which means that you can use the same methods (`invoke`, `stream`, `get_state`, etc.) in your development and production environments. This page describes how to initialize a `RemoteGraph` and interact with it.
```

Comment:
`/oss/python/langgraph/graph-api#compiling-your-graph` returns a 404 page in the preview.

Really weird, but when I click the link, the URL contains an extra `/python` in the path, which is causing the 404 page. Maybe this is just a bug in the preview.